cloudify_marketplace/resources/cloudify_security_reconfiguration

```diff
@@ -0,0 +1,267 @@
+#! /usr/bin/env python
+import json
+import logging
+import logging.handlers
+import os
+import re
+import shutil
+import subprocess
+import time
+import yaml
```

Comment:
(PEP8 says:
standard library imports
related third party imports
local application/library specific imports)

And I'm not sure on cloudify stuff being seen as 3rd party. I mean, technically yes, but would any external user see it that way?